code/_expts/slotnick_etal_2001/analyze/stats/de_StatsGroupBasicsSlot.m

```diff
@@ -0,0 +1,82 @@
+function stats = de_StatsGroupBasicsSlot( mSets, ms, ss )
+
+dss = {'train','test'};
+for dsi=1:length(dss)
+    ds = dss{dsi};
+    
+    
+    %%%%%%%%
+    % Must match # of instances across both experiments
+    %%%%%%%%
+    cate = ss.cate.rej.cc;
+    coord = ss.coor.rej.cc;
+    % Find # instances to use
+    nInstF = [ size(cate.perf.(ds){1},1), ...
```

Comment:
Kitterle. I changed them to be Cat and Coord since that's what I did for most of the other variables. Also replaced some return vars with ~ (the ones I'm not using of course).